.github/workflows/master.yml

```diff
@@ -0,0 +1,50 @@
+name: CI for master branch
+
+on:
+  push:
+    branches: master
+
+jobs:
+  push_images:
+    name: Push Docker images
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - fdb-kubernetes-operator
+          - foundationdb-kubernetes-sidecar
+          - fdb-data-loader
```

Comment:
I think the only image we want to publish here is the operator. The sidecar is normally published as part of the FDB release process, since it has to be versioned with FDB. We don't regularly publish the data loader image, but I guess there's no harm in it unless it confuses people.

Reply:
I removed the sidecar but I think it makes sense to publish the data-loader as part of the release.